src/components/Canary/CanaryStatusChart/index.js

```diff
@@ -55,11 +55,11 @@ export function CanaryStatusChart({ check, ...rest }) {
   }
 
   // @TODO: date should be formatted properly depending on selection, not just by DD/MM
-  let formatDate = (date) => dayjs(date).format("HH:mm");
+  let customFormatDate = (date) => formatDate(date, "HH:mm");
```

Comment:
formatDate should not accept a format - formats can only be defined in date.ts, and they must be consistently used. 
In this case relativeDate is probably better

Reply:
I am defining a enum for the date format types which will be consistent and is going to have 3-4 types which will be used throughout the application